build-tools/utils.sh

```diff
@@ -23,8 +23,13 @@ case $(uname -m) in
   *)      fail "unsupported architecture: $(uname -m)"
 esac
 
-case "$OSTYPE" in
-  darwin*)  OS="mac" ;;
-  linux*)   OS="linux" ;;
-  *)        fail "unsupported platform: ${OSTYPE}"
-esac
+: "${OS:=unset}"
+case "$OS" in
+  unset)
```

Comment:
ok I got what's wrong: we set the OS w/ `macos`. I worked before b/c the `OS` was ignored until this change.  

```suggestion
  macos)  OS="mac" ;;
  unset)
```